.gitignore

```diff
@@ -6,3 +6,4 @@ coverage/
 /spec/reports
 /spec/support/last_execution_examples_result.txt
 .tags_sorted_by_file
+/Gemfile.lock
```

Comment:
good!!

Reply:
but I had to remove the bundle audit check from codeclimate